src/main/java/seedu/address/model/appointment/NonConflictingAppointmentList.java

```diff
@@ -47,6 +55,14 @@ public boolean hasConflict(Appointment toCheck) {
         return internalList.stream().anyMatch(toCheck::hasConflict);
     }
 
+    /**
+     * Returns true if the list contains appointments that are in conflict with {@code toCheck}
+     */
+    public boolean hasEditConflict(Appointment toCheck) {
```

Comment:
same issue here with `allMatch()`, should use `hasConflict` instead

Reply:
you are right.